wrapper/wrapper.go

```diff
@@ -50,10 +56,101 @@ type Specification struct {
 
 // Statistics holds all available statistics: network, tcp/tcp6, cgroups and filesystem
 type Statistics struct {
-	Network     []NetworkInterface             `json:"network"`
-	Connection  TcpInterface                   `json:"connection"` //TCP, TCP6 connection stats
-	CgroupStats *cgroups.Stats                 `json:"cgroups"`
-	Filesystem  map[string]FilesystemInterface `json:"filesystem"`
+	Network         []NetworkInterface             `json:"network"`
+	Connection      TcpInterface                   `json:"connection"` //TCP, TCP6 connection stats
+	CgroupStats     *cgroups.Stats                 `json:"cgroups"`
+	CgroupsExtended *CgroupExtended                `json:"cgroups_extended"`
+	Filesystem      map[string]FilesystemInterface `json:"filesystem"`
+}
+
+// CgroupsExtended holds additional group statsistics like cpuset and shares
```

Comment:
Should we leave a comment here, that these statistics are not supplied by libcontainer implementation so that you gather them here?